packages/node/src/plugins/AutomaticPublicPathPlugin.ts

```diff
@@ -1,4 +1,10 @@
-import { Compiler, RuntimeGlobals } from 'webpack';
+import { normalizeWebpackPath } from '@module-federation/sdk/normalize-webpack-path';
+import type { Compiler } from 'webpack';
+
+const { RuntimeGlobals } = require(
```

Comment:
This can work ,but i think for these stuff which webpak has exposed , we can use compiler.webpack.TARGET to get . And it can make code looks more pretty (for me :D) .  

And if for non-exposed , like `webpack/lib/runtime/StartupChunkDependenciesPlugin` , we can use normalizeWebpackPath to get .

Reply:
It's not mandatory, just personal coding style , you can ignore it :D

Reply:
Yeah i was thinking about this today, and wanted to come up with lint rules so you cant deep import things you can get off webpack main instance :P Ill fix in new PR